packages/lib/src/components/internal/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,78 @@
+import { h } from 'preact';
+import { useCallback, useMemo } from 'preact/hooks';
+import cx from 'classnames';
+import uuid from '../../../utils/uuid';
+import './Toggle.scss';
+
+interface ToggleProps {
+    label?: string;
+    labelPosition?: 'before' | 'after';
+    ariaLabel?: string;
+    description?: string;
+    checked: boolean;
+    disabled?: boolean;
+    readonly?: boolean;
+    onChange?(checked: boolean): void;
+}
+
+const Toggle = ({ label, labelPosition = 'after', ariaLabel, description, checked, disabled = false, readonly = false, onChange }: ToggleProps) => {
+    const descriptionId = useMemo(() => (description ? `toggle-description-${uuid()}` : null), [description]);
+    const computedAriaLabel = useMemo(() => ariaLabel ?? label, [ariaLabel, label]);
+
+    const conditionalClasses = cx({
+        'adyen-checkout-toggle--label-first': labelPosition === 'before',
+        'adyen-checkout-toggle--disabled': disabled,
+        'adyen-checkout-toggle--readonly': readonly
+    });
+
+    const onInputChange = useCallback(
+        (event: Event) => {
+            onChange((event.target as HTMLInputElement).checked);
+        },
+        [onChange]
+    );
+
+    return (
+        <label className={`adyen-checkout-toggle ${conditionalClasses}`}>
+            <input
+                disabled={disabled}
+                checked={checked}
+                onChange={onInputChange}
+                aria-label={computedAriaLabel}
```

Comment:
We don't need `aria-label` here. The value from the label already comes from the wrapping label.

Reply:
To be honest I am not sure - The bento definition for Toggle says that 'toggle only' is one of the possible ways of using it, although I don't see any use-case (at least for us) right now.

I added that anyway since I wanted to make the component comply with the Bento one. 
But if you think that we don't need this, then I can remove this feature